gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.12-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.12.1-bin.zip
```

Comment:
Please show me where the current Gradle docs recommend using 8.12.1 ? I don't believe this myself.

Reply:
They write this for just about every patch:

> We recommend users upgrade to 8.12.1 instead of 8.12.

https://docs.gradle.org/8.12.1/release-notes.html

Reply:
It's great that Gradle releases a patch version supposedly to fix stuff, breaks things, doesn't release a new patch 8.12.2 to fix it and recommends upgrading to it.

https://docs.gradle.org/8.12.1/release-notes.html
**We recommend users upgrade to 8.12.1 instead of 8.12.**

It's ok, I'll rollback this change, I didn't expect things to break on a minor update released a couple of months ago.  Nothing agains 8.13.0 in particular, as I've said, it should be properly tested (and now I see even patch versions should be tested).

Reply:
Yeah, the 8.12.1 break really surprised me too! AFAICT it mostly affects Gretty, but I haven't found an alternative for running GWT superDev... Gretty is used a lot by GWT-related code it seems.